src/podium/deck.py

```diff
@@ -1,5 +1,6 @@
 import os
 from urllib.parse import quote
+from ntpath import basename
```

Comment:
`os.path.basename()` would be a better option here so it isn't Windows specific (IIRC, `os.path.basename` *is* `ntpath.basename` on windows - but it's `posix.basename` on Unix and macOS)